templates/common/run/run.go

```diff
@@ -28,72 +29,75 @@ import (
 // arbitrarily.
 const DefaultRunTimeout = time.Minute
 
-// Run is a wrapper around exec.CommandContext and Run() that captures stdout
-// and stderr as strings. The input args must have len>=1.
-//
+// Simple is a wrapper around [Run] that captures stdout and stderr as strings.
 // This is intended to be used for commands that run non-interactively then
 // exit.
 //
-// This doesn't execute a shell (unless of course args[0] is the name of a shell
-// binary).
-//
-// If the incoming context doesn't already have a timeout, then a default
-// timeout will be added (see DefaultRunTimeout).
+// If the command exits with a nonzero status code, an *os.ExitError will be
+// returned.
 //
 // If the command fails, the error message will include the contents of stdout
 // and stderr. This saves boilerplate in the caller.
-func Run(ctx context.Context, args ...string) (stdout, stderr string, _ error) {
-	stdout, stderr, _, err := run(ctx, false, args...)
-	return stdout, stderr, err
-}
-
-// RunAllowNonzero is like Run, except that if the command has a nonzero exit
-// code, that doesn't cause an error to be returned.
-func RunAllowNonzero(ctx context.Context, args ...string) (stdout, stderr string, exitCode int, _ error) {
-	return run(ctx, true, args...)
+func Simple(ctx context.Context, args ...string) (stdout, stderr string, _ error) {
```

Comment:
nits: Without reading the comments, it is difficult to know what `simple` does here by reading the function name. Wondering whether `RunWithStd` is better here?

Reply:
This name follows the guideline in https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name to think about the package name when choosing the function name. Callers see this as "run.Simple", which I think is a good name.